### DISCUSSION TITLE
Calendar component opens showing dates before the minDate

### DISCUSSION THREAD

**Discussion by silvio-junior** [primefaces/community#1297] [PrimeReact] [answered]
Hey guys. I am using the calendar and I noticed that if I set the minDate as new Date("2024/05/20") for example, it will open the calendar in the current month, focusing on today. I was expecting it to open the calendar in May, focusing on the 20th, for the date I mentioned earlier. Depending on the business rules, if the minDate is months or years in the future, the user will have to navigate to that date, wasting unnecessary time. Is this the expected behavior?

**Comment by melloware** [comment, marked as answer]
You can use the `viewDate` property see my example: https://stackblitz.com/edit/h3ebhu?file=src%2FApp.jsx
